bcipy/signal/model/offline_analysis.py

```diff
@@ -114,18 +113,30 @@ def offline_analysis(data_folder: str = None,
 
     log.info(f'Training complete [AUC={model_performance.auc:0.4f}]. Saving data...')
 
-    fig_handles = generate_offline_analysis_screen(
-        data, labels, model=model, folder=data_folder,
-        down_sample_rate=downsample_rate,
-        fs=fs, save_figure=True, show_figure=False,
-        channel_names=analysis_channel_names_by_pos(channels, channel_map))
     model.save(data_folder + f'/model_{model_performance.auc:0.4f}.pkl')
-
+    visualize_erp(
+        data,
+        labels,
+        fs,
+        save_path=data_folder,
+        channel_names=analysis_channel_names_by_pos(channels, channel_map),
+        show_figure=False
+    )
+    visualize_erp(
+        data,
+        labels,
+        fs,
+        plot_average=True,
+        save_path=data_folder,
+        channel_names=analysis_channel_names_by_pos(channels, channel_map),
+        show_figure=False,
+        figure_name='average_erp.pdf'
+    )
     if alert_finished:
         offline_analysis_tone = parameters.get('offline_analysis_tone')
         play_sound(offline_analysis_tone)
 
-    return model, fig_handles
+    return model
```

Comment:
Looks like this will break the tests at:
https://github.com/CAMBI-tech/BciPy/blob/1.5.1/bcipy/signal/tests/model/test_offline_analysis.py#L37
https://github.com/CAMBI-tech/BciPy/blob/1.5.1/bcipy/signal/tests/model/test_offline_analysis.py#L56

Reply:
I left some notes on how to update the pytest-mpl tests that compare figure contents at https://github.com/CAMBI-tech/BciPy/blob/1.5.1/bcipy/signal/model/README.md 

Let me know if these notes are not clear/if you want some help updating those tests.

Reply:
Good catch! I only ran `make test-all`